sound/soc/intel/common/soc-acpi-intel-sdw-mockup-match.c

```diff
@@ -103,6 +127,15 @@ static const struct snd_soc_acpi_adr_device sdw_mockup_amp_2_group1_adr[] = {
 	}
 };
 
+static const struct snd_soc_acpi_adr_device sdw_mockup_multi_function_adr[] = {
+	{
+		.adr = 0x0000000105AAAA00ull,
```

Comment:
Do we want o make it SDCA compliant with `0x0000000105AAAA01ull` ?

Reply:
Yes, good idea.